apps/torus-allocator/src/app/components/agent-info-card.tsx

```diff
@@ -9,23 +9,23 @@ import { Copy } from "lucide-react";
 import type { Agent } from "~/utils/types";
 
 export function AgentInfoCard({ agent }: Readonly<{ agent: Agent }>) {
-  const { tokensPerWeek, usdValue, isLoading, isError } =
+  const { isLoading, isError, displayTokensPerWeek, displayUsdValue } =
     useWeeklyUsdCalculation(agent);
 
   // Error SAFE - If the data is not loaded, display a loading state
-  const displayTokensPerWeek = isLoading ? (
+  const showTokensPerWeek = isLoading ? (
     <p className="animate-pulse text-sm">Loading...</p>
   ) : isError ? (
     "-"
   ) : (
-    tokensPerWeek
+    displayTokensPerWeek
   );
-  const displayUsdValue = isLoading ? (
+  const showUsdValue = isLoading ? (
     <p className="animate-pulse text-sm">Loading...</p>
   ) : isError ? (
     "-"
   ) : (
-    usdValue
+    displayUsdValue
   );
```

Comment:
Nitpick: why not like just `: isError ? "-" : displayUsdValue;` without the parentheses?

Reply:
Ok, prettier being weird.